src/ProjectTemplates/Shared/TemplatePackageInstaller.cs

```diff
@@ -35,15 +35,6 @@ internal static class TemplatePackageInstaller
             "Microsoft.DotNet.Web.ProjectTemplates.6.0",
             "Microsoft.DotNet.Web.ProjectTemplates.7.0",
             "Microsoft.DotNet.Web.ProjectTemplates.8.0",
-            "Microsoft.DotNet.Web.Spa.ProjectTemplates.2.1",
```

Comment:
@DamianEdwards do we want to leave around everything except 8.0?

Reply:
@danroth27 your call. I would think we would remove all of these so that the 8.0 SDK doesn't carry any SPA project templates at all.

Reply:
I think we should remove them all.